api/src/main/java/jakarta/enterprise/concurrent/ContextServiceDefinition.java

```diff
@@ -23,11 +23,15 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.Target;
 
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.inject.spi.Producer;
+import jakarta.inject.Qualifier;
+
 /**
  * <p>Defines a {@link ContextService}
  * to be injected into
  * {@link ContextService} injection points
- * with the specified {@link #qualifiers()}
+ * with the specified required {@link #qualifiers()}
```

Comment:
I'm concerned the way it's written implies that having qualifiers is required, 
```suggestion
 * including any required {@link Qualifier} annotations specified by {@link #qualifiers()}
```

Reply:
This would apply to the similar usage in the other *Definition.java files as well

Reply:
Good point. I can see how that could be misinterpreted. I'll add your suggestion to clarify.